### PR TITLE
Address List Feedback

### PR DESCRIPTION
**Pull request by OR13**
Feedback gathered from https://mailarchive.ietf.org/arch/msg/cose/LCvZEcA2oeYdSDa2Lv7gp2fJIxo/

### CODE REVIEW

**Comment by OR13** [comment]
Consider attempting better CDDL by applying https://www.ietf.org/archive/id/draft-bormann-cbor-draft-numbers-01.html

File: draft-steele-cose-merkle-tree-proofs.md [hunk 1]
```diff
@@ -208,15 +208,6 @@ One example of a Signed Inclusion Proof is a "transparent statement" as defined
 ])
 ~~~~
 
-### Array form CDDL
```
Comment (OR13) [inline draft-steele-cose-merkle-tree-proofs.md:211]: I remove this array form, as it causes confusion and does not seem necessary

File: draft-steele-cose-merkle-tree-proofs.md [hunk 2]
```diff
@@ -306,7 +297,7 @@ See {{-certificate-transparency-v2}}, 2.1.3.1. Generating an Inclusion Proof.
 The cbor representation of the inclusion proof is:
 
 ~~~~ cddl
-inclusion-proof = #6.1234([
+inclusion-proof = #TBD_2([
```
Comment (OR13) [inline draft-steele-cose-merkle-tree-proofs.md:300]: any change suggestions on this CDDL very welcome.

File: draft-steele-cose-merkle-tree-proofs.md [hunk 3]
```diff
@@ -341,6 +332,17 @@ See the privacy considerations section of:
 - {{-certificate-transparency-v2}}
 - {{-COSE}}
 
+Although the word transparency implies to some degree read access,
```
Comment (OR13) [inline draft-steele-cose-merkle-tree-proofs.md:335]: I attempt to add some text about privacy here, comments welcome

File: draft-steele-cose-merkle-tree-proofs.md [hunk 4]
```diff
@@ -350,7 +352,7 @@ Tree algorithm designers are encouraged to comment on this property of their lea
 
 # Security Considerations
 
-See the privacy considerations section of:
+See the security considerations section of:
```
Comment (OR13) [inline draft-steele-cose-merkle-tree-proofs.md:355]: fix copy paste error

**Review by ad-l — APPROVED** [review]

File: draft-steele-cose-merkle-tree-proofs.md (hunk 2 above)
Comment (henkbirkholz) [reply draft-steele-cose-merkle-tree-proofs.md:300]: good enough for now

File: draft-steele-cose-merkle-tree-proofs.md (hunk 3 above)
Comment (henkbirkholz) [reply draft-steele-cose-merkle-tree-proofs.md:335]: :+1:

File: draft-steele-cose-merkle-tree-proofs.md (hunk 4 above)
Comment (henkbirkholz) [reply draft-steele-cose-merkle-tree-proofs.md:355]: :+1:

**Review by henkbirkholz — APPROVED** [review]

**Pull request merged**